pages/blog/openforce-2022.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "AsyncAPI at OpenForce"
+date: 2022-03-31T06:00:00+01:00
+type: Communication
+tags:
+  - Open Source
+  - Contribute
+cover: /img/posts/openforce-2022/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Dev Comm Keeper
```

Comment:
There's a piece of me that thinks that we should fully spell out the word community because I wonder if people could think "Comm" stands for communication 🤔

Reply:
how about now? 😄